tests/spread/extensions/snaps/glib-hello/snap/snapcraft.yaml

```diff
@@ -5,7 +5,6 @@ description: This is a basic gnome snap (*only* uses glib). It simply prints a h
 
 grade: devel
 confinement: strict
-base: core18
```

Comment:
Why drop the base?  I think in this case it's important.

Reply:
there's a generic "set-base" call under `prepare`